src/scripts/main.js

```diff
@@ -1,3 +1,172 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const employeeTable = document.querySelector('table');
+  let currentSort = { column: null, order: 'asc' };
+  let selectedRow = null;
+
+  const headers = document.querySelectorAll('th');
+
+  headers.forEach((header, index) => {
+    header.addEventListener('click', () => {
+      const order =
```

Comment:
It is a good practice to avoid using magic numbers in your code. The numbers `3` and `4` are not immediately clear in your code. It's better to assign them to a constant variable with a descriptive name, so anyone reading your code can understand what these numbers represent.